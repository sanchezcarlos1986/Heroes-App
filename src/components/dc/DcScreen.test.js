import React from 'react';
import {DcScreen} from '~components';
import {shallow} from 'enzyme';

describe('DcScreen', () => {
  let component;

  beforeEach(() => {
    component = shallow(<DcScreen />);
  });

  it('should renders ok', () => {
    expect(component).not.toBeNull();
    expect(component).toMatchSnapshot();
  });
});
