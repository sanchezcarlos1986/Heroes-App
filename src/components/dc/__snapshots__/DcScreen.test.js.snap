// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DcScreen should renders ok 1`] = `
<div>
  <h1>
    DcScreen
  </h1>
  <HeroList
    publisher="DC Comics"
  />
</div>
`;
